src/hn_dead_links/api.clj

```diff
@@ -0,0 +1,66 @@
+(ns hn-dead-links.api
+  (:require [org.httpkit.client :as http]
+            [clojure.data.json :as json]))
+
+(def base-uri "https://hacker-news.firebaseio.com/v0")
+(def max-timeout 4000)
+
+(defn get-request
+  [url]
+  @(http/get url))
```

Comment:
Is this dereferencing necessary? Otherwise, you could just write `(def get-request http/get)` or just use http/get instead of get-request.

Reply:
This is necessary when using httpkit. httpkit is async so using `@` makes it synchronous (I think).

http://www.http-kit.org/client.html#sync

Reply:
I think that is correct to force synchronous.